src/cf-atomic-component/src/components/AtomicComponent.js

```diff
@@ -110,15 +110,17 @@ assign( AtomicComponent.prototype, Events, classList, {
     return this;
   },
 
+  /* eslint-disable complexity */
```

Comment:
I would leave the complexity warnings. They are just warnings, but are good reminders to break complex methods apart. Unless that isn't possible, which I don't think is the case here.

Reply:
I agree they are good reminders, but I've been finding they interfere with readability and seeing other issues with IDE inline linters, plus we never seem to take care of them.

<img width="527" alt="screen shot 2018-07-25 at 10 12 28 am" src="https://user-images.githubusercontent.com/1280430/43209942-7e389618-8ff3-11e8-8fd3-20b9341b6d06.png">

Maybe a good middle ground is to add a TODO that would only be highlighted on one line, and still output to the console?

Reply:
Shoot, probably shoulda @anselmbradford mentioned you.

Reply:
If the issue is dev IDE configuration preference, could the bothersome rules be disabled in the IDE linter (https://github.com/AtomLinter/linter-eslint/pull/666)? Otherwise, the `TODO` seems the way to go.

Reply:
For new code, the linting errors are important, it's far easier to avoid writing in an error from the start than editing something that's existed or written by someone else, so I'm hesitant to kill the option entirely, otherwise I'd say lets just turn it off for the entire project. I'll update to TODOs and hopefully some day they'll be fixed 😜